### ISSUE TITLE
Build schema as files

### ISSUE THREAD

**Issue by sandeepkumardev** [graphile/postgraphile#1012]
I saw it was possible to generate schema for usage.
but I do have an existing database which has a fair amount of data and I use postgraphile as a middleware on it. Is there a way to make changes in database in application level.
i.e add a column to existing table in postgraphile instead of doing in the existing database.
Also is it possible to build the schema of an existing database into files, so I could control it easily and rerun the schema changes whenever it is required.

**Comment by benjie** [comment]
I’m not sure I understand your query, please could you rephrase it? Are you trying to make changes to the database schema, or to your GraphQL schema? If the latter, check out:

https://www.graphile.org/postgraphile/make-extend-schema-plugin/

Does your later question relate to database migrations?

**Comment by sandeepkumardev** [comment]
I am referring to the first one. database migrations.

let me give you an example. Previously I was using sequelize and was able to do the migrations from there. 
but now in postgraphile i don't see this option and I have to do the migrations manually i.e change in the database directly.

Is there a way to overcome the manual work by doing it in postgraphile.

**Comment by benjie** [comment]
Sure, PostGraphile is unopinionated on migrations - you can use whatever tool you want for that. You can continue to use Sequelize, or use Sqitch or migra or apgdiff or db-migrate or pg-migrate or any of the plethora of migration tools - use whatever works for you.

**Comment by benjie** [comment]
[semi-automated message] Thanks for your question; hopefully we're well on the way to helping you solve your issue. This doesn't currently seem to be a bug in the library so I'm going to close the issue, but please feel free to keep requesting help below and if it does turn out to be a bug we can definitely re-open it 👍
